pages/voting-guides/index.tsx

```diff
@@ -0,0 +1,149 @@
+import { useState, useEffect } from "react";
+import { NextPage } from "next";
+import Head from "next/head";
+import { 
+  Scalars,
+  useUpsertVotingGuideMutation,
+  useVotingGuidesByUserIdQuery,
+  VotingGuideResult } from "generated";
+import { Layout, Avatar, FlagSection, Button } from "components";
+import { useAuth } from "hooks/useAuth";
+import styles from "./VotingGuides.module.scss";
+
+
+const VotingGuideCard = ({ guide }:{ guide: VotingGuideResult }) => {
+  const {user} = guide;
+  const {firstName, lastName, avatarUrl, username} = user;
+  const name = firstName ? `${firstName} ${!!lastName ? lastName:''}`: username;
+  return(
+  <div className={`${styles.roundedCard} ${styles.votingCardContainer}`}>
+  <div className={styles.guideContainer}>
+    <div className={styles.avatarContainer}>
+      <Avatar src={avatarUrl || ""} size={80} fallbackSrc="https://www.gravatar.com/avatar/" alt={name} />
+      <h4>{name}</h4>
+    </div>
+    <div className={styles.buttonWrapper}>
+      <Button large primary theme="blue" label="Edit">Edit</Button>
+      <span className={styles.shareButton}>
+        <Button large primary theme="yellow" label="Share">Share</Button>
+      </span>
+    </div>
+  </div>
+</div>);
+};
+
+interface VotingGuideElection {
+  date: string,
+  description?: string,
+  guides: VotingGuideResult[],  
+  id: Scalars['ID'],
+  title: string,
+}
```

Comment:
is it possible to use a generated type here?

Reply:
I could, if we link a list of guides for each election.